dask_jobqueue/tests/test_jobqueue_core.py

```diff
@@ -42,6 +43,24 @@ def test_command_template():
         assert ' --preload mymodule' in cluster._command_template
 
 
+@pytest.mark.parametrize('Cluster', [PBSCluster, MoabCluster, SLURMCluster,
+                                     SGECluster, LSFCluster, OARCluster])
+def test_shebang_settings(Cluster):
+    default_shebang = "#!/usr/bin/env bash"
+    python_shebang = "#!/usr/bin/python"
+    with Cluster(cores=2, memory='4GB', shebang=python_shebang) as cluster:
+        job_script = cluster.job_script()
+        assert job_script.startswith(python_shebang)
+        assert 'bash' not in job_script
+    with Cluster(cores=2, memory='4GB') as cluster:
+        job_script = cluster.job_script()
+        assert job_script.startswith(default_shebang)
+
+    with pytest.raises(ValueError, match="batch script interpreter"):
+        with (dask.config.set(shebang=None)):
```

Comment:
Sorry.  My example code was wrong.  `dask.config.set('jobqueue.%s.shebang' % Cluster.scheduler_name, None)` should work.

Reply:
thanks, my initial idea was way worse.